### DISCUSSION TITLE
Sonoff DW2 WiFi...

### DISCUSSION THREAD

**Discussion by homelink21** [RavenSystem/esp-homekit-devices#1528] [MEPLHAA Scripts]
Is it possible to "homekitize" the Sonoff DW2 door / window sensor? If yes, someone could help me to define a suitable json configuration for this sensor. Thanks

**Comment by nickforev** [comment]
> Is it possible to "homekitize" the Sonoff DW2 door / window sensor? If yes, someone could help me to define a suitable json configuration for this sensor. Thanks

Hi. its not possible, Sonoff DW2 runs on the OPL1000 chip instead of an ESP8266.

**Comment by homelink21** [comment]
Ok, thank’s. Are there other door sensor (such as Tuya or others) that can be converted to homekit?
